api/spec/v1.yaml

```diff
@@ -1614,6 +1681,71 @@ components:
       description: |
         A ParaTime block.
 
+    RuntimeEventList:
+      type: object
+      required: [events]
+      properties:
+        events:
+          type: array
+          items:
+            $ref: '#/components/schemas/RuntimeEvent'
+      description: |
+        A list of runtime events.
+
+    RuntimeEvent:
+      type: object
+      required: [round, type, body]
+      properties:
+        round:
+          type: integer
+          format: int64
+          description: The block height at which this event was generated.
+          example: *runtime_block_round_1
+        tx_hash:
+          type: string
+          nullable: true
+          description: |
+            Hash of this event's originating transaction
+            Absent if the event did not originate from a transaction.
+          example: *tx_hash_1
+        type: 
+          description: The type of the event.
+          $ref: '#/components/schemas/RuntimeEventType'
+        body:
+          type: object
+          description: |
+            The decoded event contents. This spec does not encode the many possible types;
+            instead, see [the Go API](https://pkg.go.dev/github.com/oasisprotocol/oasis-sdk/client-sdk/go/modules).
+            This object will conform to one of the `*Event` types two levels down
+            the hierarchy, e.g. `MintEvent` from `Event > accounts.Event > MintEvent`,
+            OR `evm.Event` directly.
+        evm_log_name:
+          type: string
+          nullable: true
+          description: |
+            If the event type is `evm.Log`, this field describes the type of evm event, e.g. `Transfer`.
+            Absent if the event type is not `evm.Log`.
+          example: 'Transfer'
+        evm_log_params:
+          type: object
+          description: |
+            The decoded `evm.Log` event data. We currently support only two types of evm events, ERC20 `Transfer` 
+            and `Approve`.
+            Absent if the event type is not `evm.Log`.
```

Comment:
nit here and elsewhere: We have `evm.log` (lowercase) in the enum. I'm fine with changing either. I'm low-key regretting one of the enums I made in the past that's all-lowercase, looks a little weird ... OTOH I was following the JSON names of fields in the top-level `Event` struct. Whatever, as long as we're consistent, I guess.

Reply:
got it, updated!